oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -155,20 +155,20 @@ private ComputeEngineCredentials(ComputeEngineCredentials.Builder builder) {
   /** Clones the compute engine account with the specified scopes. */
   @Override
   public GoogleCredentials createScoped(Collection<String> newScopes) {
-    ComputeEngineCredentials.Builder builder =
-        this.toBuilder().setHttpTransportFactory(transportFactory).setScopes(newScopes);
-    return new ComputeEngineCredentials(builder);
+    return createScoped(newScopes, null);
```

Comment:
nit: Perhaps use `ImmuntableList.of()` or `ImmutableSet.of()` instead of null for the second param?